sections/product.liquid

```diff
@@ -356,6 +356,7 @@
   const variants = JSON.parse(variantStr.slice(1, -1));
   const ratings = "{{ 'reviews.ratings' | t }}";
   const productId = '{{ selected_product.id }}';
+  const sizeBigMessage = "{{ 'general.size_big_message' | t }}";
```

Comment:
Could you rephrase `"sizeBigMessage" `to mean an image of significant size or something similar? like largeImgSize